src/Tribe/Plugins_List.php

```diff
@@ -0,0 +1,126 @@
+<?php defined( 'WPINC' ) or die;
+
+if (! class_exists( 'Tribe__Plugins_List' ) ) {
+	/**
+	 * A list of Tribe's major plugins. Useful when encouraging users to download one of these.
+	 */
+	class Tribe__Plugins_List {
```

Comment:
Can we avoid this class as a whole by inserting our plugins on the `__construct` of the _Dependendecy_ class?